app/views/layouts/application.html.erb

```diff
@@ -10,13 +10,15 @@
 
   <body>
     <nav class = "topnav">
-      <%= link_to "All Merchants", "/merchants"%>
+      <%= link_to "All Merchants", "/merchants" unless current_admin? %>
```

Comment:
This will route admins to the admin merchant index where they can activate or deactivate merchants

Reply:
Very sleek!